gen_cert.py

```diff
@@ -517,7 +518,7 @@ def _generate_certificate(
         achievements_paragraph = u"{0}{1}".format(achievements_string, achievements_description_string)
 
         # Overide achievements/interstitial strings with yaml designations info based on designation
-        designation = self.designation
+        designation = designation or self.designation
```

Comment:
Let's start with just adding the log statements (we should add one here too),
to see what's actually there.
`self.designation` should be set via the constructor.